packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -74,6 +74,7 @@ export class DataStores implements IDisposable {
             (id: string, createParam: CreateChildSummarizerNodeParam)  => CreateChildSummarizerNodeFn,
         private readonly deleteChildSummarizerNodeFn: (id: string) => void,
         baseLogger: ITelemetryBaseLogger,
+        private readonly metadata: IContainerRuntimeMetadata,
```

Comment:
Doesn't look like this needs to be a class field, can just be a param (i.e. drop the `private readonly`)